matlab_simulation/04-sensor/scanner_bresenham/scanner_bres.m

```diff
@@ -0,0 +1,75 @@
+clear;clc;
```

Comment:
I'm not sure why this file now exists. You example.m file appears to do the same thing and does a pretty good job. Perhaps this can be removed.